src/app/components/Share/index.lazy.scss

```diff
@@ -15,3 +15,118 @@
     color: $color-white-transparent-75;
   }
 }
+
+
+.isFuture {
```

Comment:
I think this class is added by PL. I've applied the `.is-future` class to the root element Odyssey cares about. I think it'd be better to use that for consistency with what I've already done.